<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PDWIVEDI" name="com.sapphire.etl.AERS_ISRID" tests="7" failures="0" timestamp="18 Jun 2014 08:59:32 GMT" time="0.389" errors="1">
  <testcase name="AERSIndicationISRIDTest" time="0.030" classname="com.sapphire.etl.AERS_ISRID"/>
  <testcase name="AERSDrugISRIDTest" time="0.030" classname="com.sapphire.etl.AERS_ISRID"/>
  <testcase name="AERSOutcomeISRIDTest" time="0.029" classname="com.sapphire.etl.AERS_ISRID"/>
  <testcase name="AERSSourceISRIDTest" time="0.033" classname="com.sapphire.etl.AERS_ISRID">
    <error type="java.lang.AssertionError" message="SELECT count(*) FROM aers_report_source where isr_id is null or isr_id = &#039;&#039;">
      <![CDATA[java.lang.AssertionError: SELECT count(*) FROM aers_report_source where isr_id is null or isr_id = ''
	at org.testng.Assert.fail(Assert.java:83)
	at com.sapphire.etl.AERS_ISRID.AERSSourceISRIDTest(AERS_ISRID_NotNull.java:168)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.AssertionError: Lists differ at element [0]: 0 != 1 expected [0] but found [1]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:549)
	at org.testng.Assert.assertEquals(Assert.java:514)
	at com.sapphire.etl.AERS_ISRID.AERSSourceISRIDTest(AERS_ISRID_NotNull.java:163)
	... 24 more
]]>
    </error>
  </testcase> <!-- AERSSourceISRIDTest -->
  <testcase name="AERSReactionISRIDTest" time="0.026" classname="com.sapphire.etl.AERS_ISRID"/>
  <testcase name="AERSDemoISRIDTest" time="0.215" classname="com.sapphire.etl.AERS_ISRID"/>
  <testcase name="AERSTherapyISRIDTest" time="0.026" classname="com.sapphire.etl.AERS_ISRID"/>
</testsuite> <!-- com.sapphire.etl.AERS_ISRID -->
